### PR TITLE
Make clear what a boolean argument corresponds to.

### PR DESCRIPTION
**Pull request by bangerth**
While looking into follow-ups to #17699.

### CODE REVIEW

**Comment by bangerth** [comment]
The failing test is the usual `trilinos/solver_control_06`, which is unrelated to this patch.

File: source/base/mpi.cc [hunk 1]
```diff
@@ -713,7 +713,11 @@ namespace Utilities
       // processes again and extended with information with the actual owner
       // of that the index.
       internal::ComputeIndexOwner::ConsensusAlgorithmsPayload process(
-        owned_indices, indices_to_look_up, comm, owning_ranks);
+        owned_indices,
+        indices_to_look_up,
+        comm,
+        owning_ranks,
+        /* keep track of requesters = */ false);
```
[the remark this answers is not in the document]
Comment (bangerth) [reply source/base/mpi.cc:720]: The parameter is poorly named then. What we really do keep track of is *who* (=which other process) is requesting. So I think that the comment here is correctly spelled.

[the remark this answers is not in the document]
Comment (masterleinad) [reply source/base/mpi.cc:720]: We should just rename the parameter as well then.

**Comment by bangerth** [comment]
Good idea. I renamed the function argument and corresponding member variable.

**Review by masterleinad — APPROVED** [review]

**Pull request merged**